runtimes/peregrine/src/lib.rs

```diff
@@ -772,47 +780,132 @@ impl InstanceFilter<Call> for ProxyType {
 			ProxyType::Any => true,
 			ProxyType::NonTransfer => matches!(
 				c,
-				Call::System(..) |
```

Comment:
I though it would be best to make all allowed calls explicit, and everything else is denied by default. The reverse could cause unauthorised calls to be allowed, while in this way potentially authorised calls might be denied, which is a safer approach.

Reply:
I would assume the same.